graphics/nxwidgets/UnitTests/CButton/cbuttontest.cxx

```diff
@@ -64,9 +64,9 @@
 
 CButtonTest::CButtonTest()
 {
-  m_bgWindow = (CBgWindow *)NULL;
-  m_nxFont   = (CNxFont *)NULL;
-  m_text     = (CNxString *)NULL;
+  m_bgWindow = NULL;
+  m_nxFont   = NULL;
+  m_text     = NULL;
```

Comment:
just for my understanding. a generic question not related to this PR. shouldn't cxx code use `nullptr`?

Reply:
nullptr doesn't exist before c++11, NULL will be defined to nullptr after c++11.